okapi-core/src/test/java/org/folio/okapi/util/DepResolutionTest.java

```diff
@@ -220,23 +235,180 @@ public void test4(TestContext context) {
     });
   }
 
+  // install optional with no provided ingerface enabled
+  @Test
+  public void testInstallOptional1(TestContext context) {
+    Async async = context.async();
```

Comment:
In all these tests there is no need to use Async.
Replace "res -> { context.assertTrue(res.succeeded());" by "context.asyncAssertSuccess(res ->"
and "res -> { context.assertTrue(res.failed());" by "context.asyncAssertFailure(exception ->"

Reply:
oh yes.. Seen that before

Reply:
Can't do like that, cause AsyncResult != ExtendedAsyncResult. I know one could use a utility or something, but least the async is systematically used.

Reply:
That's true and technical dept. https://issues.folio.org/browse/OKAPI-810 "Replace ExtendedAsyncResult by AsyncResult where possible"